setup.py

```diff
@@ -21,15 +21,22 @@
 ]
 
 install_requires=[
-    'stanford-stk @ git+https://github.com/stanford-futuredata/stk.git@main',
-    'grouped_gemm @ git+https://github.com/tgale96/grouped_gemm@main',
-    'mosaicml-turbo==0.0.4',
+    "triton==2.1.0",
+    "stanford-stk>=0.0.6",
 ]
 
 extra_deps = {}
 
-extra_deps['dev'] = [
-    'absl-py',
+extra_deps["gg"] = [
+    "grouped_gemm @ git+https://github.com/tgale96/grouped_gemm@main",
```

Comment:
We should use a tagged version instead of main branch

Reply:
I'm moving this package onto PyPi so we can push the new megablocks packages.